README.md

```diff
@@ -18,17 +18,59 @@ Some examples for the planed progression:
 - Botania -> Astral Sorcery -> Embers
 - Hearth Well + Tinkers Construct -> TC Smeltery
 - Storage Drawers + OC + IE -> Refined Storage (without auto crafting) + OC Late Game -> Applied Energistics (with auto crafting)
-- Immersive Engineering + Embers + OpenComputers -> *Every other tech mod*
-- Unique Crops + Botania + Astral Sorcery -> *Every other magic mod*
+- Immersive Engineering + Embers + OpenComputers -> _Every other tech mod_
+- Unique Crops + Botania + Astral Sorcery -> _Every other magic mod_
 
 ## Installation
 
-To play the modpack just use CurseForge version: https://www.curseforge.com/minecraft/modpacks/craftblock. Alternatively, you can find the release zip files also in [releases](https://github.com/sebinside/CraftBlock/releases).
+To play the modpack just use CurseForge version: <https://www.curseforge.com/minecraft/modpacks/craftblock>. Alternatively, you can find the release zip files also in [releases](https://github.com/sebinside/CraftBlock/releases).
 
 This can be installed using the [CurseForge-Launcher](https://download.curseforge.com/) (bad) or [MultiMC](https://multimc.org/) (better).
 
 In order to use the current version of the **override files** or to help in development, just clone and override the files in the modpack folder afterwards.
 
+### Self-hosted server
+
+#### With [Java](https://java.com) installed on host
+
+You can find a `CraftBlock-Server-vX.Y.Z.zip` in [releases](https://github.com/sebinside/CraftBlock/releases). Extract the zip file into an empty folder.\
+Execute the `startserver`-script (`.bat` on Windows / `.sh` on Unix) to set everything up and start the server on your local machine. The script will ask you to accept the EULA of Mojang.
+
+#### With [Docker](https://docker.com)
+
+`docker-compose.yml` (ensure to use the latest [release](https://github.com/sebinside/CraftBlock/releases) in `CF_SERVER_MOD`):
+
+```yml
+version: "3.9"
+
+services:
+  craftblock:
+    container_name: craftblock
+
+    image: itzg/minecraft-server:java8
+
+    stdin_open: true # docker run -i
+    tty: true # docker run -t
+
+    restart: unless-stopped
+
+    environment:
+      # EULA: "TRUE"
+      TYPE: "CURSEFORGE"
+      CF_SERVER_MOD: "https://github.com/sebinside/CraftBlock/releases/download/v1.4.0/CraftBlock-Server-v1.4.0.zip"
```

Comment:
here should be no concrete version number. i will certainly not udpate this number every time :D

Reply:
Understandable, but how else can this be solved?
GitHub has no link to a latest release asset, or similar.
I also added a note regarding this in line [41](https://github.com/sebinside/CraftBlock/pull/93/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R41).

I myself have developed a [tool](https://github.com/voruti/GithubLatestAsset) which can solve this, but it would require a server to host this program; e.g.: https://github.redno.de/sebinside/craftblock/Server